timur/docker-compose.yml

```diff
@@ -0,0 +1,62 @@
+version: "3.4"
+volumes:
+  app-gems:
+    driver: local
+  timur-db-data:
+    driver: local
+  timur-node-modules:
+    driver: local
+  timur-log-dir:
+    driver: local
+  timur-tmp-dir:
+    driver: local
+  timur-public-dir:
+    driver: local
+
+x-timur_base: &timur_base
+  build:
+    context: .
+    dockerfile: docker/app/Dockerfile.development
+  volumes:
+    - .:/app:cached
+    - app-gems:/app/vendor/bundle:rw
+    - ./node_modules:/app/node_modules:cached
+    - timur-log-dir:/app/log:rw
+    - timur-tmp-dir:/app/tmp:rw
+    - timur-public-dir:/app/public:rw
+    - ../etna:/etna:cached
+  environment: &app_environment
+    COVERAGE: "true"
+    DATABASE_HOST: "timur_db"
+    TIMUR_ENV: "development"
+
+services:
+  timur_app:
+    <<: *timur_base
+    ports:
+      - 3002:3000
+    command: ./docker/app/puma.sh
+    depends_on:
+      - timur_db
+
+  timur_webpack:
+    <<: *timur_base
+    environment:
+      <<: *app_environment
+      RUN_NPM_INSTALL: "true"
+    command: >
+      sh -c "./bin/timur create_routes && npm run webpack"
```

Comment:
For Timur, we have to run the `create_routes` command to get the latest `routes.js` file on the UI side, before calling webpack. Note that this requires us to install the Ruby gems in the Webpack container, which is different than the other ones.

Reply:
Yup, sounds good!